crates/bevy_render/src/camera/active_cameras.rs

```diff
@@ -52,11 +56,18 @@ pub fn active_cameras_system(
     query: Query<(Entity, &Camera)>,
 ) {
     for (name, active_camera) in active_cameras.cameras.iter_mut() {
-        if active_camera.entity.is_none() {
-            for (camera_entity, camera) in query.iter() {
-                if let Some(ref current_name) = camera.name {
-                    if current_name == name {
-                        active_camera.entity = Some(camera_entity);
+        match active_camera.entity {
```

Comment:
This does force us to "skip a frame" to find the "next" camera when a camera is despawned.

Can we first check if the current camera exists, if it doesn't set it to none, _then_ iterate the query to find the camera if its still none?

Reply:
Done :+1: